prereise/gather/hiflddata/README.md

```diff
@@ -0,0 +1,29 @@
+# HIFLDParser
+
+### Data Source
+- Transmission Line:
+https://hifld-geoplatform.opendata.arcgis.com/datasets/electric-power-transmission-lines
+- Substation:
+https://hifld-geoplatform.opendata.arcgis.com/datasets/electric-substations
+- PowerPlant:
+https://hifld-geoplatform.opendata.arcgis.com/datasets/power-plants
+
+
+### Sample Output Data
+- https://zenodo.org/record/3530898
+
+
+### What it does
+Move the code from private GitHub proj to PreREISE. 
```

Comment:
This line is unnecessary for this README.

Reply:
will remove